### DISCUSSION TITLE
Overview of the coding standards tooling available to WordPress developers

### DISCUSSION THREAD

**Discussion by ryanwelcher** [WordPress/developer-blog-content#160] [Topic Ideas]
It would be great to go over the coding standard tools that are available to WP developers to be able to keep their code standards compliant and consistent.

The article will cover setting up locally to use:
* WordPress Coding Standards for PHP
* The tools provided by the `@wordpress/scripts` package
* Some IDE configuration - probably focus on VSCode and PHP Storm but open other or maybe it's better to be editor agnostic?

**Comment by marybaum** [comment]
Love this! Could also incorporate code suggestions and shortcuts (I'm looking at you, phpStorm) that help real devs cut down on typing and give hookers/string-patchers like me a choice of valid options we can see at a glance will give us what we're looking to do when we haven't read the code reference ;-D

**Comment by bph** [comment]
A great hook would also be the update [WordPressCS 3.0.0 is now available](https://make.wordpress.org/core/2023/08/21/wordpresscs-3-0-0-is-now-available/)

**Comment by bph** [comment]
Topic was approved in today's Editorial Group meeting. The discussion is locked. Please continue conversation on the issue

- #167
